core/src/mledb/mledb-match/mledb-match.service.ts

```diff
@@ -156,4 +168,171 @@ export class MledbMatchService {
             skillGroup: match.skillGroup.profile.description,
         };
     }
+
+    async markSeriesNcp(seriesId: number, isNcp: boolean, winningTeamName?: string): Promise<string> {
+        const r = Math.floor(Math.random() * 10000);
+        this.logger.debug(`(${r}) begin markSeriesNcp: seriesId=${seriesId}, isNcp=${isNcp}, winningTeam=${winningTeamName}`);
+
+        // const series = await this.ss.getSeriesById(seriesId, ["fixture.home",
+        // "fixture.away"]);
+        const series = await this.seriesRepo.findOneOrFail({
+            where: {
+                id: seriesId,
+            },
+            relations: ["fixture.homeName", "fixture.awayName"],
+        });
+
+        const seriesType: SeriesType = series.fixture ? SeriesType.Fixture : SeriesType.Scrim;
+
+        // winningTeam needs to be 'let' here, since we change it to FA for scrims
+        let winningTeam = await this.teamRepo.findOne({
+            where: {
+                name: winningTeamName,
+            },
+        });
+
+        if (seriesType === SeriesType.Fixture) {
```

Comment:
Wouldn't make a new enum type just for a single if/else.

Reply:
Only skimming through this; but unsure what the purpose of this is here; when we're also looking at `series.fixture` to identify if it is a fixture